plugins/woocommerce/src/Blocks/Installer.php

```diff
@@ -33,11 +33,11 @@ public function install() {
 	 */
 	public function create_pages( $pages ) {
 
-		if ( '<!-- wp:shortcode -->[woocommerce_cart]<!-- /wp:shortcode -->' === $pages['cart']['content'] ) {
+		if ( isset( $pages['cart']['content'] ) && '<!-- wp:shortcode -->[woocommerce_cart]<!-- /wp:shortcode -->' === $pages['cart']['content'] ) {
```

Comment:
Please use [the null coalescing operator](https://www.php.net/manual/en/language.operators.comparison.php#language.operators.comparison.coalesce) instead for better code readability:

```php
if ( '<!-- wp:shortcode -->[woocommerce_cart]<!-- /wp:shortcode -->' === ( $pages['cart']['content'] ?? null ) ) {
```